conf.yaml

```diff
@@ -671,7 +671,7 @@ targets:
       - glovoapp.com
       - api.glovoapp.com
     twitter: "@Glovo_ES"
-  Uber:
+  Kali Linux:
```

Comment:
You need to add a new entry, pointing to the Kali page instead of changing the name of Uber.